docs/reproducing-results.md

```diff
@@ -45,11 +45,11 @@ documentation to learn how to make inference more efficient.
    trtllm-build \
       --checkpoint_dir /tmp/trtllm \
       --output_dir /code/openmath-mistral-7b-trtllm \
-      --gpt_attention_plugin bfloat16 \
-      --gemm_plugin bfloat16 \
-      --context_fmha "enable" \
-      --max_input_len 4096 \
-      --max_output_len 512 \
+      --gpt_attention_plugin bfloat16
+      --use_paged_context_fmha enable
+      --max_input_len 3584 \
+      --max_seq_len 4096 \
+      --max_num_tokens 4096 \
```

Comment:
Is this right? Isn't `max_num_tokens` = `max_input_len + max_seq_len `

Reply:
No, max_num_tokens is actually more like batch size. It's the maximum number of generation tokens the engine can do at the same time (across all prompts it's working on). Not really sure why we need to specify both that and the batch size, but somehow that's the current way to do it in trtllm. That being said, these particular parameters are certainly not optimal, but probably good enough and close to what we actually used with prior trtllm version